administrator/components/com_joomgallery/controllers/maintenance.php

```diff
@@ -30,9 +30,9 @@ public function __construct()
     parent::__construct();
 
     // Access check
-    if(!JFactory::getUser()->authorise('core.admin', _JOOM_OPTION))
+    if(!JFactory::getUser()->authorise('core.manage', _JOOM_OPTION))
     {
-      $this->setRedirect(JRoute::_($this->_ambit->getRedirectUrl(''), false), 'You are not allowed to configure this component', 'notice');
+      $this->setRedirect(JRoute::_($this->_ambit->getRedirectUrl(''), false), 'You are not allowed to manage this component', 'notice');
```

Comment:
It would be nice to have a language constant here.

Reply:
There are similar notices thrown the following in other controllers. Shouldnt we add the language constants for all of them in another PR? I mean it is not really related to the access rule of the maintenance view.
- administrator/components/com_joomgallery/controllers/config.php (Line 35)
- administrator/components/com_joomgallery/controllers/cssedit.php (Line 44)
- administrator/components/com_joomgallery/controllers/maintenance.php (Line 35)

Reply:
Language constants would also have to be added in some other places, for example in the upload.
I would also be in favour of doing this in a separate PR for all places.

Reply:
Does a not given answer here mean that you do not want to consider my suggestion?